server/server/session/src/test/java/com/alipay/sofa/registry/server/session/providedata/PushEfficiencyImproveServiceTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.registry.server.session.providedata;
+
+import static org.mockito.Mockito.mock;
+
+import com.alipay.sofa.registry.common.model.ServerDataBox;
+import com.alipay.sofa.registry.common.model.constants.ValueConstants;
+import com.alipay.sofa.registry.common.model.metaserver.ProvideData;
+import com.alipay.sofa.registry.server.session.push.ChangeProcessor;
+import com.alipay.sofa.registry.server.session.push.FirePushService;
+import com.alipay.sofa.registry.server.session.push.PushProcessor;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author jiangcun.hlc@antfin.com
+ * @since 2023/2/22
+ */
+public class PushEfficiencyImproveServiceTest extends PushEfficiencyImproveService {
```

Comment:
Why does the test extend the class to test?

Reply:
No special meaning, only convenient to write unit test

Reply:
This is not a good practice, I would suggest not extending the class to test.

Reply:
ok, let me adjust